pegasus/sites.v3/hourofcode.com/public/index.haml

```diff
@@ -14,6 +14,10 @@ social:
 -# whenever we change the hoc_mode, to make sure we're still testing what we'll see on production.
 -hoc_mode = DCDO.get("hoc_mode", CDO.default_hoc_mode)
 
+-# Spanish(Mexico), Spanish(Spain), and Portuguese(Brazil)
```

Comment:
Is it intentional that we're showing Latin American content for a European locale?

Reply:
Yes, per a request from Suky/Hadi we want all Spanish locales to see this video.